docs/contributing/getting-started-developers.md

```diff
@@ -135,6 +137,26 @@ After starting the client using `yarn start` in VSCode Debug View (`Ctrl + Shift
 - [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
 - [Debugging in Visual Studio Code](https://code.visualstudio.com/docs/editor/debugging)
 
+## Troubleshooting
+You may encounter some issues when using or developing in WebTS. It could be due to folder permissions, missing to create the Template packages,... Find below a list of folders that you may be interested in visiting and removing if this happens to you.
+
+You need permissions to write on the following folder, as `WebTS` creates a *WebTS* folder there. A `Logs` and `Templates` folder will be created here when running `Web Template Studio`.
+
```

Comment:
We should add some info about the template cache too. 

The template cache is located in the **.templateengine** folder. Template caches are isolated by environment.